ServerCore/Pages/Shared/_RoleSwitcherPartial.cshtml

```diff
@@ -8,19 +8,17 @@
         var IsAdmin = EventRole == ModelBases.EventRole.admin || await EventModel.IsEventAdmin();
         var IsAuthor = EventRole == ModelBases.EventRole.author || await EventModel.IsEventAuthor();
 
-        if (IsAdmin || IsAuthor)
+        var IsRegistered = await EventModel.IsRegisteredUser();
```

Comment:
I'm not sure what this is trying to achieve. If the player is not an author or admin we should not show the switcher because there is nothing to switch to. Is there a specific bug you are trying to fix with this?